Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM ruby:2.6.5
```

Comment:
This is different to what is in .ruby-version, but we should use 2.6.5. I've tested it against 2.6.5, so we can either update .ruby-version in this PR, or I can make another one

Reply:
It says 2.6.5 in here: https://github.com/dxw/affordable-housing-monitoring/blob/develop/.ruby-version

Reply:
Doh, so it does. Ignore me!

Reply:
😬